### ISSUE TITLE
Merge filters (at least manually) is broken

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/quickFilters#41]
After the last step of filter merge from the filter list toolbar - (selecting the target filter an pressing [Merge] - there is no next step:

![image](https://user-images.githubusercontent.com/6314674/104458801-be8eec80-55a3-11eb-8892-f4ffd9537614.png)

Nothing happens in the UI after pushing the [Merge] button.

In error console we see the following output:

```
12:49:13.929 Uncaught TypeError: actions.enumerate is not a function
    merge chrome://quickfilters/content/qFilters-list.js:422
    oncommand chrome://messenger/content/FilterListDialog.xhtml:1
    readMailFromURI chrome://pep4tb/content/pEpForThunderbird.js:352
    processNewMessage chrome://pep4tb/content/pEpForThunderbird.js:454
    msgAdded chrome://pep4tb/content/pEpForThunderbird.js:418
qFilters-list.js:422:29
```

**Comment by RealRaven2000** [comment]
Some rewrites for Thunderbird 78:

[quickFilters-wx-5.1pre4.zip](https://github.com/RealRaven2000/quickFilters/files/5808901/quickFilters-wx-5.1pre4.zip)

to install, download zip file and drag into Add-ons manager.

**Comment by RealRaven2000** [comment]
Fixed in 5.1 released on 27/01/2021
